NitroxPatcher/Patches/CyclopsShieldButton_StartShield_Patch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Core;
+
+namespace NitroxPatcher.Patches
+{
+    class CyclopsShieldButton_StartShield_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(CyclopsShieldButton);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("StartShield", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        public static void Postfix(CyclopsShieldButton __instance)
+        {
+            string guid = GuidHelper.GetGuid(__instance.subRoot.gameObject);
+            // Shield is activated, if activeSprite is set as sprite
+            NitroxServiceLocator.LocateService<Cyclops>().BroadcastChangeShieldState(guid, (__instance.activeSprite == __instance.image.sprite));
```

Comment:
minor: a local for readability would be good here:

```
boolean isShieldActive = (__instance.activeSprite == __instance.image.sprite);
```

same in the below patch